libraries/AP_RangeFinder/AP_RangeFinder_HC_SR04.cpp

```diff
@@ -34,10 +34,24 @@
 
 extern const AP_HAL::HAL& hal;
 
+const AP_Param::GroupInfo AP_RangeFinder_HC_SR04::var_info[] = {
+
+    // @Param: SPD_SCAL
+    // @DisplayName: Ultrasound speed scaling factor for HC-SR04
+    // @Description: Speed scaling factor of the propogating waves based on the environment. This is approximately equal to 2/(speed of sound in the environment)
+    // @Range: 10 300
+    // @User: Advanced
+    AP_GROUPINFO("SPD_SCAL", 1, AP_RangeFinder_HC_SR04, speed_scale, 58.0f),
```

Comment:
the '1' needs to not collide with the Wasp backend, this should be 15 for example